examples/tests/sdk_tests/bucket/public_url.w

```diff
@@ -0,0 +1,18 @@
+bring cloud;
+
+let bucketProps = cloud.BucketProps{public: true};
+let publicBucket = new cloud.Bucket(bucketProps) as "publicBucket";
+let privateBucket = new cloud.Bucket() as "privateBucket";
+
+test "publicUrl" {
+  publicBucket.put("file1.txt", "Foo");
+  privateBucket.put("file2.txt", "Bar");
+
+  assert(publicBucket.publicUrl("file1.txt") != "");
+
+  try {
+    privateBucket.publicUrl("file2.txt");
+  } catch e {
+    assert(e == "Cannot provide public url for a non-public bucket");
+  }
+}
```

Comment:
good!
(soon we'll have fetch and then we can test if by sending a GET request to the public URL, we actually get the value)
I'll open an issue to add to this one :)